### PR TITLE
Plain js with object

### PR DESCRIPTION
**Pull request by Apatakiti**
In this project, we have done the following:
- Created a collection that keeps a list of books.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection.
- Displayed all books saved in the collection in the top part of the page.
- Made sure that when a user clicks the "Add" button: A new book is added to the collection.
- The new book was displayed on the page.
- Made sure that when a user clicks the "Remove" button: The correct book was removed from the collection.
- The correct book disappeared from the page.
- Made sure that data is preserved in the browser's memory by using localStorage
- Remembered to remove debugging code before pushing your code to the remote repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,55 +2,60 @@
...
 ### Setup
 
-### Install
+1. Create an empty folder in your computer to store the clone.
 
-### Usage
+2. Copy the following link and paste it into your VSCode or your terminal:
 
-### Run tests
+> git@github.com:angeluray/awesome-books.git
```
Comment (julie-ify) [inline README.md]: - [x] I don't think this is your github link for cloning the repo. Kindly fix that.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  booksContainer.innerHTML = '';
+  for (let i = 0; i < bookDatas.length; i += 1) {
+    booksContainer.innerHTML += `<div class="book">
+                     <div>${bookDatas[i].title}</div>
+                     <div>${bookDatas[i].author}</div>
+                     <button class="rem-btn" onclick="removeBook(${i})">Remove</button>
+                     <hr><br></div>`;
+  }
+}
+
```
Comment (julie-ify) [inline app.js]: - [ ] On page reload, your app should fetch the books from the local storage and display them on the page. But that is not so. Nothing gets displayed on the page. Kindly fix that.

  sav-e

File: README.md (hunk 1 above)
Comment (julie-ify) [inline README.md:33]: - [ ] I don't think this is the link for cloning the repo. Kindly fix that.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Ademola-coding @RodChak,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Apatakiti) [reply README.md]: Done

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Ademola-coding 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**